client/src/components/SearchResult.js

```diff
@@ -180,10 +180,11 @@ export const RequestRequirements = ({ resource }) => {
 
   const requirements = []
   const MTA = 'Material Transfer Agreement'
-  if (resource.needs_abstract) requirements.push('Abstract')
-  if (resource.needs_irb) requirements.push('IRB')
-  if (resource.needs_mta) requirements.push(MTA)
-  if (resource.needs_shipping_info) requirements.push('Shipping Information')
+  // if (resource.needs_abstract) requirements.push('Abstract')
+  // if (resource.needs_irb) requirements.push('IRB')
```

Comment:
Any particular reason for these to hang around?

Reply:
I didn't see them in the response yet I didn't know if I am waiting for an attachment for them or a different attribute name

Reply:
Oh, looks like they're just missing from the elasticsearch view. I just push an UNTESTED :scream: commit to address that. Can you test it? (I'm working on another branch with migrations right now.)

Reply:
yeah, ill update the testData too